site/content/docs/5.2/examples/cards/index.html

```diff
@@ -93,89 +95,88 @@ <h6 class="card-text">Caption</h6>
         </div>
       </div>
 
-      <div class="col m-0 py-2">
+      <div class="col">
         <div class="card border-0">
           {{< placeholder width="100%" height="162" class="card-img-top" text="Image cap" >}}
           <div class="card-body bg-dark">
-            <h4 class="card-title">Title</h4>
-            <h6 class="card-text">Caption</h6>
+            <h5 class="card-title">Title</h5>
+            <p class="card-subtitle">Caption</p>
```

Comment:
If it is a `.card-subtitle` shouldn't we have a `<h*>` instead of a `<p>`?

Reply:
Yes, I just asked to the design and it's a short description of the title so it definitely should be `.card-text .fw-bold .mb-2`. Changing it in the next commit !